.flowconfig

```diff
@@ -1,3 +1,8 @@
+[options]
+suppress_comment=\\(.\\|\n\\)*\\$ExpectError
+suppress_comment=\\(.\\|\n\\)*\\$FlowFixMe
```

Comment:
I changed `suppress_comment` to use `$ExpectError` to match the practices of `flow-typed`. I had to add back the `$FlowFixMe` suppression comment. I felt it was better to show the intent of the errors a bit better.

Reply:
Yep, that's perfect